### DISCUSSION TITLE
RedeemerPointer 'publish' meaning

### DISCUSSION THREAD

**Discussion by mirceahasegan** [CardanoSolutions/ogmios#361] [Q & A] [answered]
Hello,
The [conway.cddl](https://github.com/IntersectMBO/cardano-ledger/blob/master/eras/conway/impl/cddl-files/conway.cddl#L477) mentions this set of redeemer tags:
```
redeemer_tag =
    0 ; Spending
  / 1 ; Minting
  / 2 ; Certifying
  / 3 ; Rewarding
  / 4 ; Voting
  / 5 ; Proposing
```

The [Ogmios schema](https://github.com/CardanoSolutions/ogmios/blob/v6.0/clients/TypeScript/packages/schema/src/index.ts#L993) has a slightly different one:

```
export interface RedeemerPointer {
  purpose: "spend" | "mint" | "publish" | "withdraw";
  index: UInt64;
}
```

Does `publish` pointer in ogmios schema refer to `Certifying` from the cddl?

Thank you!

**Comment by KtorZ** [comment]
> Does publish pointer in ogmios schema refer to Certifying from the cddl?

Correct.

**Reply by mirceahasegan** [reply, marked as answer]
So `Voting` and `Proposing` are not mapped yet. Correct?
